src/app/containers/DropDown/DropDownInfoString.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
```

Comment:
Thats not wrong, but i dont like the style to import the default and a named export

more beautiful is 

```javascript
import React from 'react'

export class Component extends React.Component
```
or 
```javascript
import { default as React, Component } from 'react'
```
funny fact: no one knows, that `default as` is possible 🤓